src/devices/configured-device-card.ts

```diff
@@ -130,15 +140,57 @@ class ESPHomeConfiguredDeviceCard extends LitElement {
             @action=${this._handleOverflowAction}
           >
             <mwc-icon-button slot="trigger" icon="more_vert"></mwc-icon-button>
-            <mwc-list-item>Validate</mwc-list-item>
-            <mwc-list-item>Install</mwc-list-item>
-            <mwc-list-item>Show API key</mwc-list-item>
-            <mwc-list-item>Download YAML</mwc-list-item>
-            <mwc-list-item>Rename</mwc-list-item>
-            <mwc-list-item>Clean Build Files</mwc-list-item>
-            <mwc-list-item>Delete</mwc-list-item>
+            <mwc-list-item graphic="icon"
+              >Validate<esphome-svg-icon
+                slot="graphic"
+                .path=${mdiCheck}
+              ></esphome-svg-icon
+            ></mwc-list-item>
+            <mwc-list-item graphic="icon"
+              >Install<esphome-svg-icon
+                slot="graphic"
+                .path=${mdiAppleKeyboardCaps}
+              ></esphome-svg-icon
+            ></mwc-list-item>
+            <mwc-list-item graphic="icon"
+              >Show API Key<esphome-svg-icon
+                slot="graphic"
+                .path=${mdiKey}
+              ></esphome-svg-icon
+            ></mwc-list-item>
+            <mwc-list-item graphic="icon"
+              >Download YAML<esphome-svg-icon
+                slot="graphic"
+                .path=${mdiCodeBraces}
+              ></esphome-svg-icon
+            ></mwc-list-item>
+            <mwc-list-item graphic="icon"
+              >Rename<esphome-svg-icon
+                slot="graphic"
+                .path=${mdiRenameBox}
+              ></esphome-svg-icon
+            ></mwc-list-item>
+            <mwc-list-item graphic="icon"
+              >Clean Build Files<esphome-svg-icon
+                slot="graphic"
+                .path=${mdiLayersOff}
+              ></esphome-svg-icon
+            ></mwc-list-item>
+            <li divider role="separator"></li>
+            <mwc-list-item class="warning" graphic="icon"
+              >Delete<esphome-svg-icon
+                class="warning"
+                slot="graphic"
+                .path=${mdiDelete}
+              ></esphome-svg-icon
+            ></mwc-list-item>
```

Comment:
Can we render this a bit nicer. Like this:
```suggestion
            <mwc-list-item class="warning" graphic="icon">
             <esphome-svg-icon
                class="warning"
                slot="graphic"
                .path=${mdiDelete}
              ></esphome-svg-icon>
              Delete
            </mwc-list-item>
```

Reply:
prettier formatted the code this way (I have FOrmat on Save in VS Code enabled). I think the checks won't pass if I change formatting, but I can try that.